src/entity/triangularMesh.cpp

```diff
@@ -0,0 +1,19 @@
+#include <entity/triangularMesh.h>
+
```

Comment:
```suggestion
#include <entity/triangularMesh.h>
#include <assert.h>

```

Reply:
Adicionar esse `#include <assert.h>` **apenas** caso for utilizado asserts no código. Adicionei a sugestão aqui pois numa sugestão mais abaixo eu faço uso.